### PR TITLE
Init of std libs: strings + cvs

### PR DESCRIPTION
**Pull request by trumae**

### CODE REVIEW

File: interative/interative.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 	"strings"
 
 	"github.com/chzyer/readline"
+	"github.com/rumlang/rum/libs"
```
Comment (felipeweb) [inline interative/interative.go]: lib is better for package name

File: libs/csv.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package libs
+
+import (
+	"encoding/csv"
+
+	"github.com/rumlang/rum/runtime"
+)
+
+//LoadCSV load the library to cvs processing
+func LoadCSV(ctx *runtime.Context) {
```
Comment (felipeweb) [inline libs/csv.go]: csv must be a pkg  not a file and this function can be init

File: libs/libs.go [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package libs
+
+import "github.com/rumlang/rum/runtime"
+
+//LoadStdLib the basic lib into Context
+func LoadStdLib(ctx *runtime.Context) {
```
Comment (felipeweb) [inline libs/libs.go]: do not need this function if move strings and csv to pkgs and load in init function

File: interative/interative.go (hunk 1 above)
Comment (avelino) [reply interative/interative.go]: example: https://github.com/rumlang/encoding

**Comment by marioidival** [comment]
This PR helped #99 to be done.

**Pull request closed without merging**